Extension/src/LanguageServer/extension.ts

```diff
@@ -541,8 +541,9 @@ function onResetDatabase(): void {
     clients.ActiveClient.resetDatabase();
 }
 
-function selectDefaultCompiler(): void {
-    clients.ActiveClient.promptSelectCompiler(true);
+function selectDefaultCompiler(sender?: any): void {
+    logForUIExperiment("CompilerSelect", sender);
```

Comment:
I believe this feature is part of a different experiment.  This function does the logging for the language status experiment, so you'll need to set up a new one (look at the existing one for information).